packages/generator/tests/acceptance/test_non_streaming_measurement_client.py

```diff
@@ -65,6 +65,7 @@ def test___measurement_plugin_client___measure___returns_output(
         enum_out=EnumInEnum.BLUE,
         enum_array_out=[EnumInEnum.RED, EnumInEnum.GREEN],
         protobuf_enum_out=ProtobufEnumInEnum.BLACK,
+        double2darray_out=None,
```

Comment:
Please test this parameter with a value other than None.

Reply:
Refer to this [comment](https://github.com/ni/measurement-plugin-python/pull/1020#discussion_r1871044195)